easybuild/easyconfigs/m/MICOM/MICOM-0.33.2-foss-2023b.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PythonBundle'
+
+name = 'MICOM'
+version = '0.33.2'
+
+homepage = 'https://github.com/micom-dev/micom'
+description = "Python package to study microbial communities using metabolic modeling."
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('hatchling', '1.18.0'),
+    ('SWIG', '4.1.1'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('SciPy-bundle', '2023.11'),
+    ('COBRApy', '0.29.0'),
+    ('scikit-learn', '1.4.0'),
+    ('SymEngine-python', '0.11.0'),
+    ('PyYAML', '6.0.1'),
+    ('coverage', '7.4.4'),
+    ('HiGHS', '1.7.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('qdldl', '0.1.7.post0', {
+        'checksums': ['f346a114c8342ee6d4dbd6471eef314199fb268d3bf7b95885ca351fde2b023f'],
+    }),
+    ('osqp', '0.6.5', {
+        'checksums': ['b2810aee7be2373add8b6c0be5ad99b810288774abca421751cb032d6a5aedef'],
+    }),
+    ('highspy', '1.7.0', {
```

Comment:
It would be better if we can get the python bindings for HiGHS into the HiGHs package.
Do you think you can make that happen?